libs/api/domains/fishing-license/src/lib/fishing-license.service.ts

```diff
@@ -17,12 +18,24 @@ export class FishingLicenseService {
     private logger: Logger,
   ) {}
 
+  // Returns enum value equivalent to the given fishing type
+  getLicenseCode = (licenseCode?: string | null) => {
```

Comment:
Because in some cases I need to get license code using a value straight from Fiskistofa's API service, and they return a string value back from the API. I don't want to type cast their response because I don't want to rely on them giving us correct codes.